Core/src/org/sleuthkit/autopsy/centralrepository/ingestmodule/IngestModule.java

```diff
@@ -155,6 +156,7 @@ public ProcessResult process(AbstractFile af) {
 
     @Override
     public void shutDown() {
+        IngestEventsListener.disableCentralRepositoryModule();
```

Comment:
I believe that this method should be called after the  is enabled check, since the module may never have run and corresponding counter increment may never have happened. 

Note that this code in the startUp and shutDown methods has been modified by both Ann and myself, so this code no longer auto merges. You will need to merge in the latest code in develop.